test/extensions/filters/http/lua/lua_integration_test.cc

```diff
@@ -24,20 +24,16 @@ class LuaIntegrationTest : public testing::TestWithParam<Network::Address::IpVer
         new FakeUpstream(0, FakeHttpConnection::Type::HTTP1, version_, timeSystem()));
     fake_upstreams_.emplace_back(
         new FakeUpstream(0, FakeHttpConnection::Type::HTTP1, version_, timeSystem()));
+    // Create the xDS upstream.
+    fake_upstreams_.emplace_back(
+        new FakeUpstream(0, FakeHttpConnection::Type::HTTP2, version_, timeSystem()));
```

Comment:
This is not needed right now (and all of the xDS stuff)? Can you remove for now?

Reply:
Ok.